packages/rum-react/src/domain/error/errorBoundary.ts

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import type { ErrorInfo } from 'react'
+import { addReactError } from './addReactError'
+
+interface Props {
+  fallback: Fallback
+  children: React.ReactNode
+}
+
+export type Fallback = (parameters: { error: Error; resetError: () => void }) => React.ReactNode
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Maybe we could support also Component for fallback.

Reply:
Done! As a function is also a Component, accepting any component works for both use cases (`fallback={() => ...}` and `fallbacx={Component}`)